pkg/config/config.go

```diff
@@ -785,12 +785,6 @@ func (c *RuntimeConfig) Validate(systemContext *types.SystemContext, onExecution
 		logrus.Infof("The manage-ns-lifecycle option is being deprecated, and will be unconditionally true in the future")
 	}
 
-	if c.UIDMappings != "" && c.ManageNSLifecycle {
-		return errors.New("cannot use UIDMappings with ManageNSLifecycle")
```

Comment:
I think there are some unit tests which have to be removed/adapted for those code paths.